packages/cli/src/lib/config/configErrors.spec.ts

```diff
@@ -1,15 +1,20 @@
-import {IncompatibleConfigurationError} from './configErrors';
-
-jest.mock('semver');
+const mockedCoerce = jest.fn();
+const mockedLt = jest.fn();
+const mockedGt = jest.fn();
+jest.mock('semver', () => ({
+  ...jest.requireActual('semver'),
+  coerce: mockedCoerce,
+  lt: mockedLt,
+  gt: mockedGt,
+}));
```

Comment:
This was needed since `CliUx` needs methods from `semver`. If we mock the entire `semver` module, CliUx would crash.

Reply:
Why not mock CliUx then? 🤔

Reply:
Because we would have to mock `@oclif/core`. Which I think will be more complicated